src/gui.js

```diff
@@ -8207,9 +8207,13 @@ IDE_Morph.prototype.getURL = function (url, callback, responseType) {
                 }
             };
         }
-        // cache-control, commented out for now
```

Comment:
hmm... so reading the comment it appears that we tried this approach before but that it messed with Snap4Arduino and local robot servers. I dunno, @jguille2 can you and @cycomachead  decide whether this is safe to pull? If so I'll be happy to. Thanks!

Reply:
I don't know what the issues were with robot servers, but the line `if (url.match(/^https/)) {` should present this from being applied to them.

Reply:
Ah, good point, Michael. Thanks!